phantoms/MR_XCAT_qMRI/sim_ivim_sig.py

```diff
@@ -4,6 +4,10 @@
 import json
 import argparse
 import os
+
+import sys
+sys.path.append('C:/TF_IVIM_OSIPI/TF2.4_IVIM-MRI_CodeCollection')
```

Comment:
This is your local path but we want to avoid this in the repository because we want to keep it general. It's guaranteed to fail for anyone not on windows, for example.